app/components/project-post-list.js

```diff
@@ -1,25 +1,88 @@
 import Ember from 'ember';
 
+/**
+  `project-post-list` provides a list of posts within the project and allows filtering and paging.
+
+  ## Default usage
+  ```Handlebars
+    {{project-post-list posts=posts project=project}}
+  ```
+  ## Adding post filtering
+    These additional properties allow for filtering.
+  ```Handlebars
+    {{project-post-list
+      isFiltered=isFiltered
+      isFilteringClosedPosts=isFilteringClosedPosts
+      isFilteringOpenPosts=isFilteringOpenPosts
+      filterByStatus="filterByStatus"
+      filterByType="filterByType"
+      pageNumber=page
+      posts=model
+      project=project
+      removeTypeFilter="removeTypeFilter"
+      selectedTypes=selectedTypes
+    }}
+  ```
+
+  @class Component
+  @module project-post-list
+  @extends Ember.Component
+ */
+
 export default Ember.Component.extend({
   classNames: ['project-post-list'],
 
+  /**
+    @property store
+    @type Ember.Service
+   */
   store: Ember.inject.service(),
 
+  /**
+    Returns information about the posts in order to sort and display posts in
+    the control.
+
+    @property meta
+    @type
```

Comment:
@rileytaylor looks like this here is just a regular `Object`.
<img width="643" alt="code_corps___build_a_better_future_" src="https://cloud.githubusercontent.com/assets/5448834/18375000/176d2b4c-760f-11e6-8e36-0f0ba3338d1b.png">